css/style.css

```
@@ -0,0 +1,31 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 20px;
+  line-height: 1.5;
+}
+
+header {
+  font-size: 20px;
+  margin: 0 auto;
+  text-align: center;
+  background-color: lightgreen;
```

```suggestion
  background-color:  #90ee90;
```
Perhaps, it would be better to use colour codes either RGB or hexadecimal. 

Why? Different browsers may not agree on what some color names mean. There are not names for all 16 million 24-bit colors. In fact there are only 17 W3C-standard color names. 
For more insight into this research, please check out [w3 article 1](https://www.w3.org/TR/CSS2/syndata.html#value-def-color) and [w3 article 2](https://www.w3.org/TR/css-color-3/)